java/tools/checkstyle/README.md

```diff
@@ -0,0 +1,70 @@
+# Checkstyle integration
```

Comment:
Do we need to check all these checkstyle files in? Is it possible to include it in bazel without checking it in?

Reply:
Did you mean to delete the README.md file?

Reply:
He meant importing the whole checkstyle tool as a bazel lib, instead of copying these files.

Reply:
yes, that's correct :)